public/js/Constants.js

```diff
@@ -14,8 +14,18 @@ var Constants: {
 
   resourceSize: number,
   numResources: number,
-
```

Comment:
keep lines in between to differentiate different types of constants (resource constants are different than history constants, etc.)